examples/hybrid/cli_options.jl

```diff
@@ -24,6 +24,20 @@ function parse_commandline()
         help = "Moisture model [`dry` (default), `equil`, `non_equil`]"
         arg_type = String
         default = "dry"
+        "--micro"
+        help = "Microphysics model [`nothing` (default), `0M`]"
+        arg_type = String
+        "--HSF"
```

Comment:
Can we have a `---forcing` default to none and make held-suarez forcing an option here?

Reply:
Yesss, thank you!